specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/LinkedService.json

```diff
@@ -4422,6 +4422,61 @@
           "description": "An ODBC connection string. Type: string, SecureString or AzureKeyVaultSecretReference.",
           "type": "object"
         },
+        "server": {
+          "type": "object",
+          "x-ms-format": "dfe-string",
```

Comment:
Sybapse does not support  x-ms-format, please remove

Reply:
done .Thanks!

Reply:
@Jingshu918